### PR TITLE
Update parallel-processing-expansion.md

### PR DESCRIPTION
**Pull request by jiaj12**
update zh doc

### CODE REVIEW

File: content/zh/docs/tasks/job/parallel-processing-expansion.md [hunk 1]
```diff
@@ -26,7 +25,7 @@ See [using Jobs in real workloads](#using-jobs-in-real-workloads) to learn about
 this pattern fits more realistic use cases.
 -->
 本任务展示基于一个公共的模板运行多个{{< glossary_tooltip text="Jobs" term_id="job" >}}。
-你可以用这种方法来并行执行批处理任务。
+您可以用这种方法来并行执行批处理任务。
```
Comment (howieyuen) [inline content/zh/docs/tasks/job/parallel-processing-expansion.md:28]: 我们一般直接翻译成“你”不用“您”
下同

**Comment by tengqm** [comment]
@jiaj12 Thanks for the contribution. Please do read the https://kubernetes.io/zh/docs/contribute/localization_zh/

File: content/zh/docs/tasks/job/parallel-processing-expansion.md [hunk 2]
```diff
@@ -273,7 +272,7 @@ First, copy and paste the following template of a Job object, into a file called
 这里的略微复杂的例子使用 [Jinja 模板语言](https://palletsprojects.com/p/jinja/)
 来生成清单，并基于清单来生成对象，每个 Job 都有多个参数。
 
-在本任务中，你将会使用一个一行的 Python 脚本，将模板转换为一组清单文件。
+在本任务中，您将会使用一个一行的 Python 脚本，将模板转换为一组清单文件。
```
Comment (tengqm) [inline content/zh/docs/tasks/job/parallel-processing-expansion.md:275]: 请改回”你“

File: content/zh/docs/tasks/job/parallel-processing-expansion.md [hunk 3]
```diff
@@ -387,7 +392,7 @@ frame.
 在此任务中，你运行一个命令通过取回 Pod 的日志来收集其输出。
 在真实应用场景中，Job 的每个 Pod 都会在结束之前将其输出写入到某持久性存储中。
 你可以为每个 Job 指定 PersistentVolume 卷，或者使用其他外部存储服务。
-例如，如果你在渲染视频帧，你可能会使用 HTTP 协议将渲染完的帧数据
+例如，如果您在渲染视频帧，你可能会使用 HTTP 协议将渲染完的帧数据
```
Comment (tengqm) [inline content/zh/docs/tasks/job/parallel-processing-expansion.md:395]: 请改回”你“

File: content/zh/docs/tasks/job/parallel-processing-expansion.md [hunk 4]
```diff
@@ -78,8 +77,8 @@ First, download the following template of a job to a file called `job-tmpl.yaml`
 {{< codenew file="application/job/job-tmpl.yaml" >}}
 
 ```shell
-# 使用 curl 下载 job-tmpl.yaml
-curl -L -s -O https://k8s.io/examples/application/job/job-tmpl.yaml
+   # 使用 curl 下载 job-tmpl.yaml
+   curl -L -s -O https://k8s.io/examples/application/job/job-tmpl.yaml
```
Comment (tengqm) [inline content/zh/docs/tasks/job/parallel-processing-expansion.md:81]: no change needed.

File: content/zh/docs/tasks/job/parallel-processing-expansion.md [hunk 5]
```diff
@@ -240,9 +239,9 @@ Processing item cherry
 ### 清理 {#cleanup-1}
 
 ```shell
-# 删除所创建的 Job
-# 集群会自动清理 Job 对应的 Pod
-kubectl delete job -l jobgroup=jobexample
+   # 删除所创建的 Job
+   # 集群会自动清理 Job 对应的 Pod
+   kubectl delete job -l jobgroup=jobexample
```
Comment (tengqm) [inline content/zh/docs/tasks/job/parallel-processing-expansion.md:244]: no change needed.

**Comment by tengqm** [comment]
pls squash commits

**Comment by jiaj12** [comment]
@tengqm already squashed

File: content/zh/docs/tasks/job/parallel-processing-expansion.md [hunk 6]
```diff
@@ -78,7 +78,7 @@ First, download the following template of a job to a file called `job-tmpl.yaml`
 {{< codenew file="application/job/job-tmpl.yaml" >}}
 
 ```shell
-# 使用 curl 下载 job-tmpl.yaml
+ # 使用 curl 下载 job-tmpl.yaml
```
Comment (tengqm) [inline content/zh/docs/tasks/job/parallel-processing-expansion.md:81]: nit, this is unnecessary

File: content/zh/docs/tasks/job/parallel-processing-expansion.md [hunk 7]
```diff
@@ -25,6 +24,7 @@ The sample Jobs process each item simply by printing a string then pausing.
 See [using Jobs in real workloads](#using-jobs-in-real-workloads) to learn about how
 this pattern fits more realistic use cases.
 -->
+
```
Comment (tengqm) [inline content/zh/docs/tasks/job/parallel-processing-expansion.md:27]: Empty lines like this is unnecessary.

**Pull request merged**